src/regular-expression-comment.ts

```diff
@@ -24,10 +24,28 @@ export default {
           const regexLine = node.loc?.start.line;
           if (regexLine) {
             const previousLine = regexLine - 1;
-
-            const hasRegexComment = comments.find(
-              (comment) => comment.loc?.start.line === previousLine || comment.loc?.start.line === regexLine,
-            );
+            const hasRegexComment = comments.find((comment) => {
```

Comment:
inconsistent camel-casing of Regex/regExp etc.  I recommend eliminating the abbreviation and just going with e.g. regularExpression